lib/client/order.rb

```diff
@@ -34,7 +41,14 @@ def get_products(order_id, params = {})
           response = client.get(uri({ order_id: order_id }, 'products'), params)
           return response.body
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order products', { order_id: order_id }, e)
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order products',
+            "Failed to get order products #{order_id}",
+            order_id,
+            { order_id: order_id },
```

Comment:
Same here. Drop `order_id` from the data, and make sure to add the errors from BC

Reply:
@mrogers-5s resolved